qlib/kernel/loader/vdso.rs

```diff
@@ -103,4 +213,9 @@ impl Vdso {
     pub fn VDSOAddr(&self) -> u64 {
         return self.read().vdsoAddr;
     }
+
+    pub fn get_symbol_page_offset(&self, name: &str) -> Option<u64> {
+        self.write().
```

Comment:
what's this ... a lock?

Reply:
https://github.com/QuarkContainer/Quark/blob/2e823f1db1da1606d024b860ec1681935f495723/qlib/kernel/loader/vdso.rs#L194
Adjusted to `read()`.